entity-framework/core/what-is-new/ef-core-6.0/whatsnew.md

```diff
@@ -2,30 +2,767 @@
 
-EF Core 6.0 is currently in development. This contains an overview of interesting changes introduced in each preview.
+EF Core 6.0 has reached release candidate quality. This page contains an overview of interesting changes introduced in this release.
 
-This page does not duplicate the [plan for EF Core 6.0](xref:core/what-is-new/ef-core-6.0/plan). The plan describes the overall themes for EF Core 6.0, including everything we are planning to include before shipping the final release.
+> [!TIP]
+> You can run and debug into the samples shown below by [downloading the sample code from GitHub](https://github.com/dotnet/EntityFramework.Docs/tree/main/samples/core/Miscellaneous/NewInEFCore6).
+
+## SQL Server temporal tables
+
+GitHub Issue: [#4693](https://github.com/dotnet/efcore/issues/4693).
+
+[SQL Server temporal tables](/sql/relational-databases/tables/temporal-tables) automatically keep track of all data ever stored in a table, even after that data has been updated or deleted. This is achieved by creating a parallel "history table" into which timestamped historical data is stored whenever a change is made to the main table. This allows historical data to be queried, such as for auditing, or restored, such as for recovery after accidental mutation or deletion.
+
+EF Core now supports:
+
+* The creation of temporal tables using Migrations
+* Transformation of existing tables into temporal tables, again using Migrations
+* Querying historical data
+* Restoring data from some point in the past
+
+### Configuring a temporal table
+
+The model builder can be used to configure a table as temporal. For example:
+
+<!--
+modelBuilder
+    .Entity<Employee>()
+    .ToTable("Employees", b => b.IsTemporal());
+-->
+[!code-csharp[SimpleConfig](../../../../samples/core/Miscellaneous/NewInEFCore6/TemporalTablesSample.cs?name=SimpleConfig)]
+
+When using EF Core to create the database, the new table will be configured as a temporal table with the SQL Server defaults for the timestamps and history table. For example, consider an `Employee` entity type:
+
+<!--
+    public class Employee
+    {
+        public Guid EmployeeId { get; set; }
+        public string Name { get; set; }
+        public string Position { get; set; }
+        public string Department { get; set; }
+        public string Address { get; set; }
+        public decimal AnnualSalary { get; set; }
+    }
+-->
+[!code-csharp[Employee](../../../../samples/core/Miscellaneous/NewInEFCore6/TemporalTablesSample.cs?name=Employee)]
+
+The temporal table created will look like this:
+
+```sql
+DECLARE @historyTableSchema sysname = SCHEMA_NAME()
+EXEC(N'CREATE TABLE [Employees] (
+    [EmployeeId] uniqueidentifier NOT NULL,
+    [Name] nvarchar(100) NULL,
+    [Position] nvarchar(100) NULL,
+    [Department] nvarchar(100) NULL,
+    [Address] nvarchar(1024) NULL,
+    [AnnualSalary] decimal(10,2) NOT NULL,
+    [PeriodEnd] datetime2 GENERATED ALWAYS AS ROW END NOT NULL,
```

Comment:
HIIDDEN should be added to make them hidden?

Reply:
Good question! I will follow up.

Reply:
Filed https://github.com/dotnet/efcore/issues/25841